plugins/withAndroidDayNightThemePlugin.js

```diff
@@ -0,0 +1,27 @@
+// Based on https://github.com/expo/expo/pull/33957
+// Could be removed once the app has been updated to Expo 53
+const {withAndroidStyles} = require('@expo/config-plugins')
+
+module.exports = function withAndroidDayNightThemePlugin(appConfig) {
```

Comment:
```suggestion
const { withAndroidStyles } = require('@expo/config-plugins');
const { version: expoVersion } = require('expo/package.json');

module.exports = function withAndroidDayNightThemePlugin(appConfig) {
  if (Number(expoVersion.split('.')[0]) >= 53) return appConfig;
```

Would something like this be a nice addition to ensure the plugin is not used beyond expo 53?

Reply:
I'll just remember to remove it when we update